modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb

```diff
@@ -36,8 +36,6 @@ def submit_form_2122
           end
 
           ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id)
-          data = power_of_attorney.form_data
-          ClaimsApi::PoaFormBuilderJob.perform_async(power_of_attorney.id) if data['signatureImages'].present?
```

Comment:
what's different about v0 from v1 that makes these removable here?

Reply:
I checked and most of the work was already done in the corresponding parent controller due to ITF so I was pleasantly surprised how little was necessary to implement this properly after tests.